src/i18n/messages/all.de.json

```diff
@@ -52,7 +52,10 @@
   "calendar": {
     "nextMonthAriaLabel": "Nächster Monat",
     "previousMonthAriaLabel": "Vorheriger Monat",
-    "todayAriaLabel": "Heute"
+    "todayAriaLabel": "Heute",
+    "i18nStrings.nextYearAriaLabel": "Nächstes Jahr",
+    "i18nStrings.previousYearAriaLabel": "Vorheriges Jahr",
+    "i18nStrings.currentMonthAriaLabel": "Kosten des aktuellen Monats"
```

Comment:
This is an incorrect translation ("Costs of the current month"). I'd suggest "Aktueller Monat" instead.

Reply:
Yes, I noticed this and opened a ticket (`D119465504`). Since these strings are not used until #1942, I think this could be still merged in order to unblock publishing that PR, but otherwise I'll do that anyway with the changes in this PR once it's ready with a couple more refactors. One way or the other, the translation fix will have to be integrated before merging #1942.